src/components/billingRecord/IFXBillingRecordList.vue

```diff
@@ -976,6 +1001,14 @@ export default {
                 {{ item.endDate | humanDatetime }}
               </span>
             </template>
+            <template v-slot:item.productUsage="{ item }">
+              <span v-if="item.productUsageLinkText" style="white-space: nowrap">
+                <a :href="item.productUsageUrl">{{item.productUsageLinkText}}</a>
+              </span>
+              <span v-else style="white-space: nowrap">
```

Comment:
And here as well. Or you can have the <span> be the same and put the `v-if` inside by wrapping the `{{ item.productUsage.id }}` in a <span>